Dockerfile

```diff
@@ -10,6 +11,9 @@ ENV PATH=${SBT_HOME}/bin:${PATH}
 COPY --from=sbt-cache /usr/local/sbt /usr/local/sbt
 COPY --from=sbt-cache /root/.ivy2 ${JENKINS_HOME}/.ivy2
 COPY --from=sbt-cache /root/.sbt ${JENKINS_HOME}/.sbt
+COPY --from=sbt-plugins-cache /usr/local/sbt /usr/local/sbt
```

Comment:
Why do we need to copy this twice?

Reply:
I don't see any reason we would need to.  I'll remove it.